samples/inputs/combo/templates/src/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { IgrCombo, ComboTemplateProps, IgrIcon, registerIcon } from "igniteui-react";
+import "igniteui-webcomponents/themes/light/bootstrap.css";
+import { Cities } from "./data";
+
+type City = {
+  name: string;
+  id: string;
+  country: string;
+};
+
+registerIcon("down", "images/down.svg", "material");
```

Comment:
move those icon registration inside the `ComboTemplates` component and use them inside a useEffect that is triggered only once